test/integration/local/payment-request/get-debts.test.js

```diff
@@ -0,0 +1,34 @@
+const { getDebts } = require('../../../../app/debt')
+const db = require('../../../../app/data')
+
+describe('Get debts test', () => {
+  let debts
+
+  beforeEach(async () => {
+    await db.sequelize.truncate({ cascade: true })
+
+    debts = {
+      frn: 1234567890,
+      reference: 'SIP00000000000001',
+      netValue: 15000
+    }
+
+    await db.debtData.create(debts)
+  })
+
+  afterAll(async () => {
+    await db.sequelize.truncate({ cascade: true })
+    await db.sequelize.close()
+  })
+
+  test('should return debt', async () => {
```

Comment:
Name could be more specific, `should return 1 quality check record` -- again teaming up with data module could help

Reply:
Updated the test names to be more specific